tests/trajectory/test_trj.py

```diff
@@ -0,0 +1,48 @@
+"""Pytest for dynsight.trajectory.Trj.init methods."""
+
+import os
+import tempfile
+from pathlib import Path
+from typing import Generator
+
+import MDAnalysis
+import pytest
+
+from dynsight.trajectory import Trj
+
+
+@pytest.fixture
+def original_wd() -> Generator[Path, None, None]:
+    original_dir = Path.cwd()
+
+    # Ensure the original working directory is restored after the test
+    yield original_dir
+
+    os.chdir(original_dir)
+
+
+# Define the actual test
+def test_output_files() -> None:
```

Comment:
Please submit an issue to expand this using fixtures and add more fail cases - not for now.